.github/workflows/tests.yaml

```diff
@@ -41,7 +33,7 @@ jobs:
       - name: "Set up env ${{ matrix.os }} 🐍"
```

Comment:
The sibling action in the other job uses a different name without the platform. Might be good to homogeneize.

Reply:
Which sibling action?

Reply:
The other setup-miniconda instance in this workflow

Reply:
ah, added the name manually